test/WebJobs.Script.Tests/Workers/Rpc/RpcFunctionInvocationDispatcherTests.cs

```diff
@@ -44,6 +45,40 @@ public async Task GetWorkerStatusesAsync_ReturnsExpectedResult()
             }
         }
 
+        [Fact]
+        public async Task WorkersStartInLinearTime()
+        {
```

Comment:
Nit: Instead of adding a new test - update `Starting_MultipleJobhostChannels_Succeeds` to verify the time?

Reply:
Is that test validating other things as well? It's nice to have specialized tests if that is that case, as it is a lot easier to troubleshoot the issue when there's a failure. Maintenance is simpler as well.